actix-session/src/cookie.rs

```diff
@@ -247,6 +254,12 @@ impl CookieSession {
         self
     }
 
+    /// Controls if the session cookie is sent for all sessions, or only sessions with data. Default is `false`.
```

Comment:
Wording:

When true, prevents adding session cookies to responses until session contains data.

Also a not about privacy compliance would be good.